hellosign_docsignature.go

```diff
@@ -127,7 +127,7 @@ func (m *Client) GetFiles(signatureRequestID, fileType string) ([]byte, error) {
 }
 
 // ListSignatureRequests - Lists the SignatureRequests (both inbound and outbound) that you have access to.
-func (m *Client) ListSignatureRequests() (*model.ListResponse, error) {
+func (m *Client) ListSignatureRequests() (*model.ListSignaturesResponse, error) {
```

Comment:
question @primba-deputy , in the interface we have query map[string]string
```	
ListSignatureRequests(ctx context.Context, query map[string]string) ([]models.EmbeddedSignature, error)
```
Are we going to pass it to the sdk?